client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -227,7 +227,12 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
       >
         {title ? <ModalTitle title={title} /> : null}
         <form
-          style={{ display: 'flex', flexDirection: 'column', flex: '1 1 auto' }}
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            flex: '1 1 auto',
+            overflow: 'auto',
```

Comment:
What difference does this make?

Reply:
Means modal contents will scroll, but the action buttons at the bottom are always visible, rather than having to scroll down to the buttons

Reply:
Thanks, sorry forgot to add a comment about this 🙏